keps/sig-node/1769-memory-manager/README.md

```diff
@@ -541,69 +542,49 @@ Yes, there is a number of Unit Tests designated for State file validation.
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  It is possible that the state file will have inconsistent data during the rollout, because of the kubelet restart, but
+  you can easily to fix it by removing memory manager state file and kubelet restart. It should not affect any running 
+  workloads.
+  
 
 * **What specific metrics should inform a rollback?**
+  None.
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  Tested it manually by replacing the kubelet binary on the node with the `Static` memory manager policy, but I failed
+  to find correct procedure how to test upgrade from 1.21 to my custom build with updated kubelet binary.
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  No.
 
 ### Monitoring Requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
+  The memory manager data will be available under pod resources API. When it configured with the static policy
+  you will see memory related data during call to the pod resources API List method under the container.
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  This does not seem relevant to this feature.
```

Comment:
Using the API and metrics, how will I know whether my numa alignment is or is not working?  The API is important for pod authors (was my intent honored) and the metrics are important for cluster-admins (am I providing the services requested).

Reply:
So we have two scenarios:
**Pod started** - in this case, you have two options to verify if containers pinned to the NUMA node
1. via pod resources API, you will need to connect to grpc socket and get information from it - https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#monitoring-device-plugin-resources
2. checking the relevant container cgroup under the node

**Pod failed to start because of admission error** to understand the reason you will need to check via pod resources API the amount of allocatable memory and memory reserved by containers.

Reply:
The detail here is good.  Please add it to PRR.  As discussed on slack, it's worth indicating the lack of visibility as an answer to the "Are there any missing metrics that would be useful to have to improve observability of this feature".